Core/tooling/build-final.js

```diff
@@ -21,43 +15,13 @@ const fixNodeImports = (src) => src
   .replace(/require\(['"]minimalistic-assert['"]\)/g, 'dereq_minimalistic_assert')
   .replace(/require\(['"]inherits['"]\)/g, 'dereq_inherits')
   .replace(/require\(['"]asn1\.js['"]\)/g, 'dereq_asn1');
-const nodeDepsSrc = fixNodeImports(fs.readFileSync(path.nodeDepsBundle).toString())
-const nodeDevDepsSrc = fixNodeImports(fs.readFileSync(path.nodeDevDepsBundle).toString())
-const nodeEntrypointSrc = fs.readFileSync(path.nodeEntrypointBundle).toString().replace("'[BUILD_REPLACEABLE_VERSION]'", 'APP_VERSION');
 
 // bare
 const bareDepsSrc = fs.readFileSync(path.bareDepsBundle).toString()
 const bareEntrypointSrc = fs.readFileSync(path.bareEntrypointBundle).toString().replace("'[BUILD_REPLACEABLE_VERSION]'", 'APP_VERSION');
 
 // final (node, bare, dev)
 
-const finalNodeSrc = `
-try {
-  /* final flowcrypt-android bundle starts here */
-  const dereq_inherits = require("util").inherits; // standard node util, not to interfere with webpack require, which cannot resolve it
-  ${fs.readFileSync(path.nativeCrypto).toString()}
-  ${nodeDepsSrc}
-  ${nodeEntrypointSrc}
-  /* final flowcrypt-android bundle ends here */
-} catch(e) {
-  console.error(e);
-}
-`;
-
-const finalNodeDevSrc = `
-try {
-  /* final flowcrypt-android bundle starts here */
-  const dereq_inherits = require("util").inherits; // standard node util, not to interfere with webpack require, which cannot resolve it
-  ${fs.readFileSync(path.beginAndroidDev).toString()}
-  ${fs.readFileSync(path.nativeCrypto).toString()}
-  ${nodeDevDepsSrc}
-  ${nodeEntrypointSrc}
-  /* final flowcrypt-android bundle ends here */
-} catch(e) {
-  console.error(e);
-}
```

Comment:
I think these may also be needed - but `beginAndroidDev` should be renamed to `beginDev` and modified instead of removed.